src/Resources/config/services/event_listener.xml

```diff
@@ -16,5 +16,9 @@
             <tag name="kernel.event_listener" event="sylius.product.post_create" method="updateIndex" />
             <tag name="kernel.event_listener" event="sylius.product.post_update" method="updateIndex" />
         </service>
+        <service id="bitbag_sylius_elasticsearch_plugin.event_listener.order_products" class="\BitBag\SyliusElasticsearchPlugin\EventListener\OrderProductsListener" public="true">
+            <argument type="service" id="bitbag.sylius_elasticsearch_plugin.refresher.resource" />
+            <argument type="string">fos_elastica.object_persister.bitbag_shop_product.default</argument>
```

Comment:
Is it working? should be service ID

Reply:
https://github.com/BitBagCommerce/SyliusElasticsearchPlugin/blob/master/src/Resources/config/services/event_listener.xml#L10

https://github.com/BitBagCommerce/SyliusElasticsearchPlugin/blob/master/src/EventListener/ResourceIndexListener.php#L40

Listners that listens for changes goes through a map, and uses keys to call apropriate reindexer, so this is service id as a string of product persister.